rpm/st2chatops.spec

```diff
@@ -21,6 +21,11 @@ Prefix:         /opt/stackstorm/chatops
 %define _rpmdir %(pwd)/..
 %define _build_name_fmt %%{NAME}-%%{VERSION}-%%{RELEASE}.%%{ARCH}.rpm
 
+%if 0%{?rhel} >= 8
+BuildRequires:  python36-rpm-macros
```

Comment:
```suggestion
```
Not needed, we don't care about python shebangs in st2chatops at all.

Reply:
so - my thinking was that the python36-rpm-macros was the package that provided  `%undefine %brp_mangle_shebangs`

Reply:
According to https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_basic_system_settings/using-python3_configuring-basic-system-settings

> The python36-rpm-macros package is required so that files with /usr/bin/python3 shebangs are automatically changed to /usr/bin/python3.6. For more information, see Section 8.4.4, “Handling hashbangs in Python scripts”. 

Safe to remove.

Reply:
done